packages/x-date-pickers/src/tests/describeValidation/describeValidation.types.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { MuiRenderResult, createRenderer } from '@mui/monorepo/test/utils/createRenderer';
+import { DateOrTimeView, DefaultizedProps } from '@mui/x-date-pickers/internals';
+
+type DescribeValidationComponentFamily =
+  | 'legacy-picker'
+  | 'new-picker'
+  | 'field'
+  | 'calendar'
+  | 'clock'
+  | 'new-static-picker'
+  | 'legacy-static-picker';
+
+export interface DescribeValidationInputOptions {
+  render: (node: React.ReactElement) => MuiRenderResult;
+  // TODO: Export `Clock` from monorepo
+  clock: ReturnType<typeof createRenderer>['clock'];
+  after?: () => void;
+  componentFamily: DescribeValidationComponentFamily;
+  views: DateOrTimeView[];
+}
+
+export interface DescribeValidationOptions
+  extends DefaultizedProps<DescribeValidationInputOptions, 'views'> {
```

Comment:
Sounds you removed the optional aspect of `view`
```suggestion
  extends DescribeValidationInputOptions {
```

Reply:
Good catch